migrations/20230626155016_get_products.sql

```diff
@@ -0,0 +1,57 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE PROCEDURE `GetProducts`(
+    IN product_name VARCHAR(255),
```

Comment:
cliend will probably send to you venueID, so you don't need check name or address